java-checks/src/main/java/org/sonar/java/checks/WeakSSLContextCheck.java

```diff
@@ -56,11 +56,11 @@ public class WeakSSLContextCheck extends IssuableSubscriptionVisitor {
     .withAnyParameters()
     .build();
 
-  private boolean projectHasJava8OrHigher;
+  private boolean projectNotSetOrHasJava8OrHigher;
```

Comment:
Not sure that "project not set" makes sense, it is the java version that is not set.
Maybe rename it to `projectHasJavaNotSetOr8OrHigher` or even something like `javaVersionNotSetOrSuperiorTo8`.

Reply:
Right. Changed to `javaVersionNotSetOr8OrHigher`.